en/appendices/3-4-migration-guide.rst

```diff
@@ -122,6 +122,18 @@ Cake\Validation\Validator
     * ``provider()``
 Cake\View\StringTemplateTrait
     * ``templates()``
+Cake\View\ViewBuilder
+    * ``templatePath()``
+    * ``layoutPath()``
+    * ``autoLayout()``
```

Comment:
I think the autoLayout method should not be listed here and should be mentioned separately. Because this is deprecated by isAutoLayoutEnabled/enableAutoLayout. I am not sure it could be grouped as getX/setX.